src/main/java/org/webjars/taglib/LocateTag.java

```diff
@@ -39,6 +39,15 @@
 	private Iterator<String> result;
 
 	/**
+	 * Whether the path we're looking for do not specify the version number.
```

Comment:
That's exactly the point here, to have version automatically added.  The reference to Spring MVC doc, was to show the desired functionality.

I'll try to show you how a meet the need for such a use case:
* Was trying to add bootswatch css's to my project using the taglib
* In the jar there is /META-INF/resources/webjars/bootswatch/3.3.5+4/cerulean/bootstrap.css, but also /META-INF/resources/webjars/bootswatch/3.3.5.+4/2/cerulean/bootstrap.css (see the /2/ part int the path, that means version 2).
* With this distribution, If I want to choose version 2 for the file there is no problem, I can specify /2/cerulean/bootstrap.css in the path so It is resolved.
* But If I want to choose version 3 (that does not include version number in path) I can't specify just /cerulean/bootstrap.css; as both match.  I could specify full path /bootswatch/3.3.5+4/cerulean/bootstrap.css but "What happens if i change version in POM?"
* I ended implementing functionality to support full path but without version info.

These can't be default functionality cause It my break backward compatibility (I'd prefer to be default but...  don't want to break others work)

PS: Description changed now!